consistent_file_size_units.patch

```diff
@@ -0,0 +1,21 @@
+# GtkFileChooserWidget3: allow to use IEC units when Caja use them (GTK_USE_IEC_UNITS=1)
+# when GTK_USE_IEC_UNITS=1 and when IEC units is checked in Display tab of caja-file-management-properties
+# https://github.com/GNOME/gtk/blob/3.24.41/gtk/gtkfilechooserwidget.c
+Index: b/gtk/gtkfilechooserwidget.c
+===================================================================
+--- a/gtk/gtkfilechooserwidget.c
++++ b/gtk/gtkfilechooserwidget.c
+@@ -5223,6 +5223,13 @@ file_system_model_set
+     case MODEL_COL_SIZE_TEXT:
+       if (info == NULL || _gtk_file_info_consider_as_directory (info))
+         g_value_set_string (value, NULL);
++      else if (g_strcmp0 (g_getenv ("GTK_USE_IEC_UNITS"), "1") == 0) {
```

Comment:
When I tested, I set `GTK_USE_IEC_UNITS=1` and used Pluma's open dialog, but it didn't work... until I realised that the code is reading Caja's preferences.

Are you OK if we make it agnostic so `GTK_USE_IEC_UNITS=1` will force IEC units regardless?

Reply:
Yes, if you prefer.